src/views/Type.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <main class="type">
+    <h1 v-if="this.error">Resource Type not found.</h1>
+
+    <h1 v-if="!this.error">{{this.resourceType.name}}</h1>
+    <div class="back-link-container">
+      <router-link to="/types">&#60; Back to Types</router-link>
+    </div>
+    <div class="preview-container">
```

Comment:
IMHO it would be nice to wrap all "main" elements that appear as one row in a `<div class="row">`.
Even though it may won't add any styling, it would make the structure of the page more clear.
(E.g. this is one row, this is another row below.)

Reply:
I would like to avoid div elements. Instead, I'd like to refactor the above to just use a section. (Did that in the next feature branch)